backend/public-graph/graph/resolver.go

```diff
@@ -1527,10 +1527,15 @@ func (r *Resolver) IsWithinQuota(ctx context.Context, productType pricing.Produc
 		return false, 1
 	}
 
+	basePrice := pricing.ProductToBasePriceCents(productType)
+	if stripePlan != privateModel.PlanTypeUsageBased {
+		basePrice = pricing.ProductToBasePriceCentsNonUsageBased(productType)
+	}
```

Comment:
does this need to be a separate function call, or could `pricing.ProductToBasePriceCents` just have the logic for non-usage-based plans? or is this because we rely on that constant return value of `ProductToBasePriceCents` elsewhere?

Reply:
good point, will combine the two functions